pkg/apis/kappctrl/v1alpha1/types_fetch.go

```diff
@@ -86,3 +87,14 @@ type AppFetchLocalRef struct {
 	// Object is expected to be within same namespace
 	corev1.LocalObjectReference `json:",inline" protobuf:"bytes,1,opt,name=localObjectReference"`
 }
+
+type AppFetchImgpkgBundle struct {
+	Image string `json:"image,omitempty"`
```

Comment:
Might be worthwhile looking into `imgpkg`'s Bundle definition here:

https://github.com/vmware-tanzu/carvel-imgpkg/blob/76031cd3e5dd4366b3d3163a8b24fcfc1f4c8f94/pkg/imgpkg/bundle/bundle.go

Reply:
A bit new to the project here and wasn't completely aware about using go modules versus calling the binary directly in this case. Disregard my comment.